lib/components/horizontal_section_episode.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+
+import '../graphql/queries/page.graphql.dart';
+
+import '../theme/bccm_colors.dart';
+import '../theme/bccm_typography.dart';
+import '../helpers/utils.dart';
+import 'section_episode_thumbnail.dart';
+
+class HorizontalSectionEpisode extends StatelessWidget {
+  final Fragment$ItemSectionItem sectionItem;
+  final Fragment$ItemSectionItem$item$$Episode episode;
+  final bool showSecondaryTitle;
+  final Size imageSize;
+  final bool isLive;
+
+  HorizontalSectionEpisode({
+    super.key,
+    required this.sectionItem,
+    required this.imageSize,
+    required this.showSecondaryTitle,
+    this.isLive = false,
+  }) : episode = sectionItem.item as Fragment$ItemSectionItem$item$$Episode;
+
+  bool get watched => episode.progress != null && episode.progress! > episode.duration * 0.9;
+
+  final bool isNewItem = false;
+
+  @override
+  Widget build(BuildContext context) {
+    final show = episode.asOrNull<Fragment$EpisodeShow>()?.season?.$show;
```

Comment:
I find this a bit scary or at least unintuitive. Can you instead change the query so that the ItemSectionItem fragment has `...EpisodeShow`?